package.json

```diff
@@ -42,13 +42,12 @@
     "vscode-languageserver-textdocument": "^1.0.0"
   },
   "devDependencies": {
-    "@types/sinon": "^10.0.0",
     "@types/tape": "^4.0.0",
     "c8": "^7.0.0",
+    "execa": "^6.0.0",
     "prettier": "^2.0.0",
     "remark-cli": "^10.0.0",
     "remark-preset-wooorm": "^9.0.0",
-    "sinon": "^12.0.0",
```

Comment:
Awesome! Mocking libraries are great, but if tests can be written without mocks, that’s even better.